tests/rados/test_bz_2213766.py

```diff
@@ -0,0 +1,138 @@
+       of mon daemon to IP address using `ceph config set` command.
+    2. Retrieve configuration value using `ceph config get` command.
+    3. Execute `cephadm-ansible` playbook to retrieve configuration
+       value.
+    4. Validate `cephadm-ansible` playbook output contains configuration value
+       from step #2.
+
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): ceph cluster
+
+    """
+    log.info("Running test for bz-1829646")
+    config = kw.get("config")
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    mgr_obj = MgrWorkflows(node=cephadm)
+    installer_node = ceph_cluster.get_ceph_object("installer")
+    ansible_dir = "/usr/share/cephadm-ansible"
+    inventory = "hosts"
+
+    try:
+
+        log.info("Retrieving active manager hostname, host object, ip ddress")
+
+        active_mgr = mgr_obj.get_active_mgr()
+        hostname = active_mgr.split(".")[0]
+        host_obj = rados_obj.get_host_object(hostname=hostname)
+        value = get_ip_from_node(host_obj)[1]
+
+        log.info(
+            f"Retrieved active manager hostname, host object and ip address\n"
+            f"Hostname {hostname}\n"
+            f"Host object {host_obj}\n"
+            f"IP address {value}\n"
+        )
+
+        who = "mgr"
+        param = f"mgr/dashboard/{hostname}/server_addr"
+
+        log.info(f"Proceeding to set Parameter {param} for {who} on host {hostname}")
+
+        if not rados_obj.set_configuration(who="mgr", param=param, value=value):
+            raise AssertionError(
+                f"Unable to set configuration for {who} param:{param} value:{value}"
+            )
+
+        log.info(
+            f"Successfully set configuration {param} for {who}\n"
+            f"Retrieving param {param} using `ceph config get`"
+        )
+
+        ceph_config_get_value = rados_obj.get_configuration(who=who, param=param)
+
+        log.info(
+            "Successfully retrieved configuration parameter value using `ceh config get`\n"
+            f"Parameter: {param}\n"
+            f"Daemon: {who}\n"
+            f"Value: {ceph_config_get_value}\n"
+            f"Retrieving configuration parameter {param} using `cephadm-ansible` playbook"
+        )
+
+        cmd = f"""cat <<EOF > {ansible_dir}/test_bz_2213766.yaml
+---
+- name: Get config paramters
+  hosts:  {value}
+  tasks:
+    - name: get the dashboard server address
+      ceph_config:
+         action: get
+         who: {who}
+         option: {param}
+EOF"""
+        rc = installer_node.exec_command(cmd=cmd, long_running=True, sudo=True)
+
+        if rc != 0:
+            raise AssertionError("ansible-playbook failed to execute")
+
+        cmd = f"cd {ansible_dir} ; ansible-playbook -v -i {inventory} {ansible_dir}/test_bz_2213766.yaml"
+        out, err = installer_node.exec_command(cmd=cmd, sudo=True)
+        if not err:
+            raise AssertionError("ansible-playbook failed to execute")
```

Comment:
is the assertion condition correct here? if Yes, please add some comments in the file as to why not having error lines is considered failure condition.

Also, if output is not too huge, can we print the console output obtained from running the playbook?